### PR TITLE
FIX: use new enable_sidebar setting

### PR DESCRIPTION
**Pull request by lis2**
Related to https://github.com/discourse/discourse/pull/17662

### CODE REVIEW

**Review by tgxworld — APPROVED** [review]

**Comment by tgxworld** [comment]
@lis2 Let's quickly merge this in but let us also follow up with a test here.

File: test/javascripts/acceptance/core-sidebar-test.js [hunk 1]
```diff
@@ -14,11 +14,12 @@ import { emojiUnescape } from "discourse/lib/text";
 import User from "discourse/models/user";
 
 acceptance("Discourse Chat - Core Sidebar", function (needs) {
-  needs.user({ experimental_sidebar_enabled: true, has_chat_enabled: true });
```
Comment (tgxworld) [inline test/javascripts/acceptance/core-sidebar-test.js:17]: Hmm is there a reason why the test didn't fail when the setting was no longer valid?

Comment (lis2) [reply test/javascripts/acceptance/core-sidebar-test.js:17]: It was failing, I just fixed it quickly to have green CI. I have to add test that old sidebar is not visible (that will be next PR)

Comment (tgxworld) [reply test/javascripts/acceptance/core-sidebar-test.js:17]: ahh ok :+1:

**Pull request merged**